From e7f88aa69e83bf4ce21e5e82b5643b6e3ec9b84c Mon Sep 17 00:00:00 2001
From: codeworkx <daniel.hillenbrand@codeworkx.de>
Date: Mon, 25 Dec 2017 17:39:18 +0100
Subject: [PATCH] bootctrl: add guard

Bootctrl isn't needed on devices which don't support
Seamless System Updates.

Change-Id: I3d905bd38e95e21ca40b8a012d6d1780d4b2568b
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index 03a11d3..49f5fe4 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(TARGET_NO_BOOTCTRL),)
 # TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
 ifneq ($(BOARD_IS_AUTOMOTIVE),true)
 ifneq ($(filter msm8996 msm8998 sdm845,$(TARGET_BOARD_PLATFORM)),)
@@ -28,3 +29,4 @@ include $(BUILD_STATIC_LIBRARY)
 
 endif
 endif
+endif
-- 
2.15.1

